examples/playground/index.tsx

```diff
@@ -104,20 +104,19 @@ const renderApp = (v: string) => {
 
   if (v === 'SSR' && appContainer && !isFailSsr) {
     const components = buildServerComponents(mode);
-    const ssr = ReactDOMServer.renderToString(
-      <App initialStep={v} components={components} />
-    );
+    const ssr = renderToString(<App initialStep={v} components={components} />);
     appContainer.innerHTML = isRender ? `<div>${ssr}</div>` : ssr;
   }
   if (v === 'PAINT LOADING') {
     const components = buildClientComponents(mode);
-    ReactDOM[isRender ? 'render' : 'hydrate'](
-      <App initialStep={v} components={components} />,
-      appContainer
-    );
+    if (isRender) {
```

Comment:
/nit 

Could keep DRY here. Maybe instantiate a variable?

```js
const reactDomRun = isRender ? render : hydrate;
reactDomRun(<App initialStep={v} components={components} />, appContainer);
```

Reply:
It's not really much more code, and find it more readable to be explicit about it in an if else personally. If the `App` props become more complicated, then I see more need for making this change.